envelope/envelope.py

```diff
@@ -1248,16 +1249,19 @@ def _gpg_verify(self, signature: bytes, data: bytes):
             fp.seek(0)
             return bool(self._gnupg.verify_data(fp.name, data))
 
-    def _get_decipherers(self) -> Set[str]:
+    def _get_decipherers(self) -> set[str]:
         """
         :return: Set of e-mail addresses
         """
         return set(x.address for x in self._to + self._cc + self._bcc + [x for x in [self._from] if x])
 
     def smime_sign_only(self, email, sign):
-        from cryptography.hazmat.primitives.serialization import load_pem_private_key, pkcs7
-        from cryptography.x509 import load_pem_x509_certificate
-        from cryptography.hazmat.primitives import hashes, serialization
+        try:
+            from cryptography.hazmat.primitives.serialization import load_pem_private_key, pkcs7
+            from cryptography.x509 import load_pem_x509_certificate
+            from cryptography.hazmat.primitives import hashes, serialization
+        except ImportError:
+            raise ImportError(smime_import_error)
```

Comment:
přesně takhle si to představuju! :) Ale aby to tam nebylo třikrát, extrahuj to do nějaké funkce jako `_import_cryptography` nebo tak